types/moment-strftime2/index.d.ts

```diff
@@ -0,0 +1,88 @@
+// Type definitions for moment-strftime2 1.0
+// Project: https://github.com/ShogunPanda/moment-strftime2
+// Definitions by: Piotr Roszatycki <https://github.com/dex4er>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+import * as _moment from 'moment';
+
+declare module "moment" {
+    interface Moment {
```

Comment:
Why do you need These two module declarations here?

Reply:
This module does monkey patching for `moment` and/or `moment-timezone` modules. It works similar to `moment-timezone` which patches `moment` module. See: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/6bd9be2e366c54038ba218c989f236759b75fd54/types/moment-timezone/index.d.ts#L11 for comparision.

Reply:
@sheetalkamat ah, about question, why both modules, not just only `moment`? Well, you can test it in `moment-strftime2-tests.ts`. Adding this interface only to first module is not enough.